src/components/Navbar/NavbarMobile.jsx

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { FiMenu, FiLogOut } from 'react-icons/fi';
+import {
+  Avatar,
+  Button,
+  Flex,
+  HStack,
+  Image,
+  Menu,
+  MenuButton,
+  MenuList,
+  MenuItem,
+  MenuDivider,
+  Spacer,
+  Text,
+} from '@chakra-ui/react';
+import NavbarLinkMobile from './NavbarLinkMobile';
+
+import logo from '../../assets/OCHlogo.png';
+
+const NavbarMobile = ({ isAdmin }) => {
+  // TO DO: get profile image and name
+  const user = {
+    firstName: 'Dan',
+    lastName: 'Abramov',
+    profilePic: 'https://bit.ly/dan-abramov',
+  };
+
+  const admin = [
+    { text: 'Monitor Logs', path: '/logs' },
+    { text: 'People', path: '/people' },
+    { text: 'Species Catalog', path: '/species' },
+    { text: 'Sections & Segments', path: '/sections' },
+    { text: 'Emergency Numbers', path: '/emergency-numbers' },
+  ];
+
+  const volunteer = [
+    { text: 'Monitor Logs', path: '/logs' },
+    { text: 'Species Catalog', path: '/species' },
+    { text: 'Sections & Segments', path: '/sections' },
+    { text: 'Emergency Numbers', path: '/emergency-numbers' },
+  ];
+
+  const adminAccountDrop = [
+    { text: 'Account', path: '/account' },
+    { text: 'Admin Portal', path: '/adminportal' },
+    { text: 'Sign Out', path: '/signout' },
+  ];
+
+  const volunteerAccountDrop = [
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'volunteerAccountDrop' is assigned a value but never used.